# Copyright (c) 2020 Robotics and Artificial Intelligence Lab, KAIST
#
# Any unauthorized copying, alteration, distribution, transmission,
# performance, display or use of this material is prohibited.
#
# All rights reserved.

float64 transition_x
float64 transition_y
float64 transition_z
float64 rotation_0
float64 rotation_1
float64 rotation_2
float64 rotation_3
float64 rotation_4
float64 rotation_5
float64 rotation_6
float64 rotation_7
float64 rotation_8
